src/components/AddProfileForm.js

```diff
@@ -0,0 +1,37 @@
+import "./AddProfileForm.css";
+
+export default function AddProfileForm() {
+  return (
+    <div className="Form-Wrapper">
+      <form className="ProfileForm">
+        <p className="ptag">Name:</p>
```

Comment:
I'd use the label tag instead of p tags, that way screen readers can recognise them

Reply:
thanks, I'll change that.